docs/source/federation/other-servers.md

```diff
@@ -5,15 +5,16 @@ sidebar_title: Third-party support
 
 The following is a list of known open source GraphQL server libraries that have built-in support for Apollo Federation.
 
-| Language    | Framework    | Library                                                                          |
-| ----------- | ------------ | -------------------------------------------------------------------------------- |
-| Go          | gqlgen       | [gqlgen](https://github.com/99designs/gqlgen/tree/master/plugin/federation)      |
-| Java        | graphql-java | [federation-jvm](https://github.com/apollographql/federation-jvm)                |
-| Kotlin      | graphql-java | [graphql-kotlin](https://github.com/ExpediaGroup/graphql-kotlin)                 |
-| PHP         | graphql-php  | [php-graphql-federation](https://github.com/pascaldevink/php-graphql-federation) |
-| PHP         | wp-graphql   | [wp-graphql-federation](https://github.com/wp-graphql/wp-graphql-federation)     |
-| Python      | ariadne      | [ariadne](https://github.com/mirumee/ariadne)                                    |
-| Python      | graphene     | [graphene-federation](https://github.com/preply/graphene-federation)             |
-| Ruby        | graphql-ruby | [apollo-federation-ruby](https://github.com/Gusto/apollo-federation-ruby)        |
+| Language    | Framework     | Library                                                                          |
+| ----------- | ------------- | -------------------------------------------------------------------------------- |
+| Go          | gqlgen        | [gqlgen](https://github.com/99designs/gqlgen/tree/master/plugin/federation)      |
+| Java        | graphql-java  | [federation-jvm](https://github.com/apollographql/federation-jvm)                |
+| Kotlin      | graphql-java  | [graphql-kotlin](https://github.com/ExpediaGroup/graphql-kotlin)                 |
+| PHP         | graphql-php   | [php-graphql-federation](https://github.com/pascaldevink/php-graphql-federation) |
+| PHP         | wp-graphql    | [wp-graphql-federation](https://github.com/wp-graphql/wp-graphql-federation)     |
+| Python      | ariadne       | [ariadne](https://github.com/mirumee/ariadne)                                    |
+| Python      | graphene      | [graphene-federation](https://github.com/preply/graphene-federation)             |
+| Ruby        | graphql-ruby  | [apollo-federation-ruby](https://github.com/Gusto/apollo-federation-ruby)        |
+| Rust        | async-graphql | [async-graphql](https://crates.io/crates/async-graphql)                          |
```

Comment:
@trevor-scheer Should we request a link to Github, not the Crate?

@sunli829 - Thank you for the PR!  We're taking a look.